src/Services/ProxyClient.php

```diff
@@ -11,47 +11,54 @@
 
 class ProxyClient extends HubspotClient
 {
-    /** @inheritDoc */
-    public function __construct($config = [], $client = null, $clientOptions = [], $wrapResponse = true)
-    {
-        if (null !== $this->client) {
-            return;
-        }
+	/** @inheritDoc */
+	public function __construct($config = [], $client = null, $clientOptions = [], $wrapResponse = true)
+	{
+		if (null !== $this->client)
+		{
+			return;
+		}
 
-        parent::__construct($config, $client, $clientOptions, $wrapResponse);
+		if (!array_key_exists('key', $config) || empty($config['key']))
```

Comment:
This will allow to use proxy without set key in config :smile: No better idea to `set` it, setting it is forced by external lib which cannot be changed

Reply:
```suggestion
		if (empty($config['key']) || !array_key_exists('key', $config))
```